ykllvmwrap/src/symbolizer.rs

```diff
@@ -63,9 +73,10 @@ mod tests {
 
     #[test]
     fn find_code_sym_mangled() {
-        let f_addr = symbolize_me_mangled as *const fn() as *const _ as usize;
+        let f_vaddr = symbolize_me_mangled as *const fn() as *const _ as usize;
+        let (obj, f_off) = code_vaddr_to_off(f_vaddr).unwrap();
         let s = Symbolizer::new();
-        let sym = s.find_code_sym(f_addr).unwrap();
+        let sym = s.find_code_sym(obj, f_off as usize).unwrap();
```

Comment:
Can the `as` be a `try_from` or similar? Ditto for the `as`'s below.

Reply:
Hrm, arguably `find_code_sym` should take a `u64` as it is dealing with a file offset, not a virtual address. Let me reshuffle some types...

Reply:
Fixed in 9433cc07425 along with some out of date commentary.